src/MgvOfferMaking.sol

```diff
@@ -242,13 +242,15 @@ contract MgvOfferMaking is MgvHasOffers {
 
       uint tickSpacing = ofp.olKey.tickSpacing;
       // normalize tick to tickSpacing
-      Bin insertionBin = BinLib.tickToNearestHigherBin(insertionTick, tickSpacing);
-      insertionTick = TickLib.fromBin(insertionBin, tickSpacing);
-      require(TickLib.inRange(insertionTick), "mgv/writeOffer/tick/outOfRange");
+      Bin insertionBin = insertionTick.toNearestBin(tickSpacing);
+      insertionTick = insertionBin.toNearestTick(tickSpacing);
```

Comment:
Reading this is weird as it looks like it's lossy in both directions. (already commented on the `toNearestTick` function, just thought I'd point out this example of why I think it should be renamed)

Reply:
fixed in 8ef390398905f50d1d11d6462ed559fe86b0a90b